openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -396,16 +397,18 @@ def _get_all_committees_from_instance(self, instance: Dict[str, Any]) -> Set[int
         Gets a Set of all committees from the instance regarding committees from group D
         """
         committees = set(instance.get("committee_ids", []))
+        if committee_str := instance.get("committee_$_management_level", {}).keys():
+            committees.update(map(int, committee_str))
 
         # In case of create there is no id, in case of update the user can remove committees only with the committee right
         if instance_user_id := instance.get("id"):
             user = self.datastore.get(
-                FullQualifiedId(Collection("user"), instance_user_id), ["committee_ids"]
+                FullQualifiedId(Collection("user"), instance_user_id),
+                ["committee_ids", "committee_$_management_level"],
             )
             committees.update(user.get("committee_ids", []))
-
-        if committee_str := user.get("committee_$_management_level", []):
-            committees.update(map(int, committee_str))
+            if committee_str := user.get("committee_$_management_level", []):
+                committees.update(map(int, committee_str))
```

Comment:
I think this is wrong: You also return all existent committee management levels here, right? Suppose user A has some rights in committee C1. I update him so that he has also rights in C2, where I have rights, but I don't have any in C1. This would currently fail, since C1 is returned here, but it should succeed, because I don't touch C1.

If this is correct, please add an explicit test for it.

Reply:
You are right, I removed getting these committee_ids. Nevertheless I added a test for it